setup.py

```diff
@@ -29,7 +29,25 @@ def read(filename):
     url='https://github.com/OPHoperHPO/image-background-remove-tool',
     packages=find_packages(),
     scripts=[],
-    install_requires=[],
+    install_requires=[
```

Comment:
We need the ability to install a different version of carvekit by extra indexes, like `carvekit[gpu]` or `carvekit[cpu]`

Reply:
Not able for now due to setuptools limitations